src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+interface Todo {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+}
+
+export const TodoInfo: React.FC<{ todo: Todo }> = ({ todo }) => {
+  const user = usersFromServer.find(user => user.id === todo.userId);
+
+  if (!user) {
+    return null; // Handle cases where the user is not found
```

Comment:
Returning `null` when the user is not found might not provide enough feedback to the user. Consider displaying a message or a placeholder indicating that the user information is missing.